frontend/appflowy_flutter/lib/plugins/database_view/application/field/field_cell_bloc.dart

```diff
@@ -30,8 +30,10 @@ class FieldCellBloc extends Bloc<FieldCellEvent, FieldCellState> {
             emit(state.copyWith(field: cellContext.field));
           },
           startUpdateWidth: (offset) {
-            final width = state.width + offset;
-            emit(state.copyWith(width: width));
+            if (state.width + offset >= 100) {
+              final width = state.width + offset;
+              emit(state.copyWith(width: width));
+            }
```

Comment:
I think 50px would be better here. Suppose I just have a checkbox

This logic works, but is not backwards compatible. Maybe you could think about what would happen if my field width was already less than 100 (or the 50 like I mentioned above) before this patch?

Reply:
Is this fine? If a user already has a field with a width less than 50px, it will become 50px when the user tries to change its width.